scripts/apps/search/directives/SearchPanel.js

```diff
@@ -306,6 +306,11 @@ export function SearchPanel($location, desks, privileges, tags, asset, metadata,
                         _.find(scope.tags.selectedFacets[type], (facet) => facet.value === key);
                 }
 
+                if (type === 'credit') {
+                    return scope.tags && scope.tags.selectedFacets[type] &&
```

Comment:
Can we just put:
```js
if (!scope.tags || !scope.tags.selectedFacets[type]) {
    return false;
}
```
on line 304 and remove it from all the other conditions? It would make things a lot more readable and less repetitive.